content/applications/websites/ecommerce/products.rst

```diff
@@ -4,275 +4,402 @@
-customer's portal.
+To make your file available (after payment), set the :guilabel:`Visibility` field to
+:guilabel:`Confirmed order` and turn off the :guilabel:`Show on product page` switch.
 
-Product configuration
-=====================
+.. _ecommerce-products-translation:
 
-Multiple languages
-------------------
+Translation
+===========
 
-If multiple languages are available on your website and you wish to have the product's information
-translated, it is necessary to encode this translated information in the **product's template**.
-Fields with multiple languages available are identifiable by their abbreviation language (ex.
-:abbr:`EN (English)`) next to their field.
+If multiple languages are available on your website, you can translate a product's information
+directly on the :ref:`product form <ecommerce-products-product-form>`. Fields that support multiple languages
+are identifiable by their abbreviation language (e.g., EN) next to their field.
 
 .. image:: products/products-field-translation.png
-   :align: center
    :alt: Field translation
 
-The **eCommerce-related** fields to translate are:
+The eCommerce-related fields to translate are:
 
 - :guilabel:`Product name`;
 - :guilabel:`Out-of-Stock Message` (under the :guilabel:`Sales` tab);
-- :guilabel:`Sales Description` (under the :guilabel:`Sales` tab);
+- :guilabel:`Sales Description` (under the :guilabel:`Sales` tab).
 
 .. note::
-   Having untranslated content on a web page may be detrimental to the user experience and
-   :doc:`SEO <../../websites/website/pages/seo>`.
-.. note::
-   To check the language(s) of your website, go to :menuselection:`Website --> Configuration -->
-   Settings --> Website Info section`.
+   - Having untranslated content on a web page may be detrimental to the user experience and
+     :doc:`SEO <../../websites/website/pages/seo>`. You can use the
+     :doc:`Translate <../website/configuration/translate>` feature to allow automatic translation of
+     standard terms and provide a tool for manual content translation.
+   - To check the language(s) of your website, go to :menuselection:`Website --> Configuration -->
+     Settings` and go to the :guilabel:`Website Info` section.
 
-.. seealso::
-   - :doc:`../../websites/website/pages/seo`
+.. _ecommerce-products-website-availability:
 
 Website availability
 --------------------
 
-A product can be set available on either *one* or *all* websites, but it is not possible to select
-*some* websites and not others. To define a product's availability, go to :menuselection:`Website
---> eCommerce --> Products`, select your product, and in the :guilabel:`Sales` tab, click the
-:guilabel:`Website` you wish the product to be available on. Leave the field empty for the products
-to be available on *all* websites.
+To set the product's availability, navigate to the :ref:`product form <ecommerce-products-product-form>`, go
```

Comment:
I know it's mentioned just above but still I think it'd be helpful to repeat it:
```suggestion
To set the product's website availability, navigate to the :ref:`product form <ecommerce-products-product-form>`, go
```